### PR TITLE
POC data migration system

### PR DESCRIPTION
**Pull request by tjerman**

### CODE REVIEW

File: pkg/app/options/corredor.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func Corredor() (o *CorredorOpt) {
 		ListRefresh:        time.Second * 5,
 		Log:                false,
 
-		TlsCertEnabled: true,
+		TlsCertEnabled: false,
```
Comment (darh) [inline pkg/app/options/corredor.go]: No!

you have  `CORREDOR_CLIENT_CERTIFICATES_ENABLED` to control this.

File: pkg/migrate/main.go [hunk 2]
```diff
@@ -0,0 +1,320 @@
+package migrate
```
Comment (darh) [inline pkg/migrate/main.go]: we have another thing we call "migrate" (for db migrations)
so, try to think of a better name .

also, how compose-oriented is this? 
we are dealing 99% with records and 1% with users, correct?
can it handle other types of data?
can it handle other formats (yaml)?

File: pkg/migrate/types/migrateable.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+var (
+	SfDateTime = "2006-01-02 15:04:05"
+)
+
+type (
+	Migrateable struct {
+		Name string
+		Path string
+
+		Source *os.File
```
Comment (darh) [inline pkg/migrate/types/migrateable.go]: how about io.Reader instead of os.File?

File: pkg/migrate/main.go (hunk 2 above)
Comment (tjerman) [reply pkg/migrate/main.go]: At the moment it's as you said; mostly Compose oriented, but.. It should be fairly trivial to extend this to an arbitrary structure and arbitrary data source

**Pull request closed without merging**